playwright/e2e/settings/encryption-user-tab/encryption-tab.spec.ts

```diff
@@ -111,4 +111,19 @@ test.describe("Encryption tab", () => {
         // The user is prompted to reset their identity
         await expect(dialog.getByText("Forgot your recovery key? You’ll need to reset your identity.")).toBeVisible();
     });
+
+    test("should warn before turning off key storage", async ({ page, app, util }) => {
+        await verifySession(app, "new passphrase");
+        await util.openEncryptionTab();
+
+        await page.getByRole("checkbox", { name: "Allow key storage" }).click();
+
+        await expect(
+            page.getByRole("heading", { name: "Are you sure you want to turn off key storage and delete it?" }),
+        ).toBeVisible();
```

Comment:
I think a screenshot can be great here to avoid regression on the "Are you sure you want to turn off key storage and delete it?" panel